target/snitch_cluster/sw/snax/streamer-simd/include/snax-streamer-simd-lib.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 KU Leuven.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Xiaoling Yi <xiaoling.yi@esat.kuleuven.be>
+
+#include <stdbool.h>
+#include "snrt.h"
+#include "stdint.h"
+
+#pragma once
+
+// the spatial unrolling of simd
+#define vec_len 64
```

Comment:
Please use capital letters for constants
```suggestion
#define VEC_LEN 64
```